Documentation/installation/k8s-install-migration.rst

```diff
@@ -193,6 +193,34 @@ Preparation
         $ cilium install |CHART_VERSION| --values values-migration.yaml --dry-run-helm-values > values-initial.yaml
         $ cat values-initial.yaml
 
+.. note::
+When you install Cilium, it will create a new network interface called ``cilium_host`` on each node. You may need to configure the old CNI to ensure that it doesn't accidentally use this interface.
```

Comment:
I don't think that this should be a callout note. It seems like instead this should be a dedicated step that starts with "Ensure that existing plugins are configured to avoid modifying Cilium-managed traffic". Then it can proceed with describing which other plugins are known to require additional steps here, then it can proceed with the rest of the existing description here to tell the user what to do for a clean migration. All of this needs to be done at a specific point in the order of steps in this document to ensure the migration is smooth.

Please also ensure that the indentation for the paragraphs is correct to make sure that the text displays correctly. This should become apparent fairly quickly in the rendered version of the text.

Reply:
Agreed.